superheroes.yaml

```
@@ -95,6 +96,40 @@ scripts:
       - abort: Opentelemetry has not been ready in 1 min
   - signal: OTEL_READY
 
+  ###### JITSERVER ######
+  start-jit-server:
+  - js: ${{JIT_SERVER_ENABLED}}
+    then:
+    - log: starting jit server...
+    - sh: >
+        ${{CONTAINER_RUNTIME}} run -d
+        --name jit-server
+        --memory ${{JIT_SERVER_MEMORY}}
+        --cpuset-cpus ${{JIT_SERVER_CPU}}
+        -p ${{JIT_SERVER_PORT}}:38400
+        ${{= getNetwork( '${{SUT_SERVER}}' ) }}
+        ${{JIT_SERVER_IMAGE:'icr.io/appcafe/ibm-semeru-runtimes:open-21-jdk-ubi9-minimal'}}
+        "jitserver" 
```

Not sure whether you want to keep this by default
```suggestion
        jitserver -XX:+JITServerLogConnections
```